components/yotpo/sources/new-reviews/new-reviews.js

```diff
@@ -0,0 +1,34 @@
+const yotpo = require('../../yotpo.app.js')
+module.exports = {
+  name: 'yotpo-new-reviews',
+  version: '0.0.1',
```

Comment:
You'll need a `key` of the format `[app slug]-[source name]`, and a `description`, which typically reads "Emits an event on each..."